recipes/r-cmprsk/meta.yaml

```diff
@@ -0,0 +1,76 @@
+{% set version = '2.2-7' %}
+
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-cmprsk
+  version: {{ version|replace("-", "_") }}
+
+source:
+  fn: cmprsk_{{ version }}.tar.gz
+  url:
+    - https://cran.r-project.org/src/contrib/cmprsk_{{ version }}.tar.gz
+    - https://cran.r-project.org/src/contrib/Archive/cmprsk/cmprsk_{{ version }}.tar.gz
+  sha256: 952da4a0fd55a039f64f619c2c7cc29ba2490985ca67d2853de511692f1fb9c6
+
+build:
+  number: 0
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+requirements:
+  build:
+    - r-base
+    - r-survival
+    - posix                # [win]
+    - {{native}}toolchain  # [win]
+    - gcc                  # [not win]
+
+  run:
+    - r-base
+    - r-survival
+    - libgcc  # [not win]
+
+test:
+  commands:
+    - $R -e "library('cmprsk')"  # [not win]
+    - "\"%R%\" -e \"library('cmprsk')\""  # [win]
+
+about:
+  home: http://www.r-project.org
+  license: GPL (>= 2)
```

Comment:
GPL requires that the license be distributed with the package. Please include a `license_file` entry.